public/locales/en/common.json

```diff
@@ -247,7 +247,7 @@
   "MembershipEnded": "Please renew Podverse Premium to use all features.",
   "MembershipInactive": "Please renew your inactive Podverse Premium to use all features.",
   "Membership preview text legend": "Click the play icon to watch a preview of the feature.",
-  "Mission statement": "Podverse creates free and open source software to expand what is possible in podcasting.",
```

Comment:
I would argue expanding what is possible in podcasting doesn't necessitate those changes being for the better.